demos/Azure/Azure-Demo.ps1

```diff
@@ -10,56 +10,56 @@
 ###
 ### Install-Package -Name AzureRM.NetCore.Preview -Source https://www.powershellgallery.com/api/v2 -ProviderName NuGet -ExcludeVersion -Destination <Folder you want this to be installed>
 ###
-### Ensure $ENV:PSMODULEPATH is updated with the location you used to install.
+### Ensure $env:PSModulePath is updated with the location you used to install.
```

Comment:
The module path environment variable on unix platforms is `PSMODULEPATH` and it's case sensitive. So maybe this should be changed to `$env:PSMODULEPATH`, to guarantee it works on all platform.